lib/generators/spotlight/install_generator.rb

```diff
@@ -21,9 +21,6 @@ def add_js_rails6
     def add_manifest
       append_to_file 'app/assets/javascripts/application.js', "\n//= require_tree .\n"
       append_to_file 'app/assets/config/manifest.js', "\n//= link spotlight/manifest.js"
-
-      # Rails installed importmap by default, but we don't have importmap + Blacklight 7 working yet.
-      remove_file 'app/javascript/application.js'
```

Comment:
Don't we need this even if we're not using the template?  The use of template.rb is not required.

Reply:
I was assuming that for fresh Rails apps it still wouldn't do much because stimulus-rails would install after. For existing apps, it seemed slightly questionable to delete their application.js (even if it results in Spotlight not working). I haven't tested those assumptions yet, so I'm happy to leave the generator untouched and work toward getting the template instructions working as written.